terraform/azure.tf

```diff
@@ -485,10 +485,10 @@ resource "random_string" "sql" {
 
 resource "azurerm_sql_server" "sql_server" {
   name                         = "sql-srv-${random_string.sql.result}"
-  resource_group_name          = azurerm_resource_group.rg.name
-  location                     = var.location
-  version                      = var.sql-server-version
-  administrator_login          = "iazAdmin"
+  resource_group_name          = "${azurerm_resource_group.rg.name}"
```

Comment:
the quotes and brackets are not required. This style is deprecated as of terraform 0.12.